README.md

```diff
@@ -72,6 +72,23 @@ To scan a directory, add the following step:
 
 The `path` key allows any valid path for the current project. The root of the path (`"."` in this example) is the repository root.
 
+## Scanning SBOM file
+
+Scan a SBOM file:
+
+```yaml
+- name: Create SBOM
+  uses: anchore/sbom-action@v0
+  with:
+    format: spdx-json
+    artifact-name: "${{ github.event.repository.name }}-sbom.spdx.json"
```

Comment:
I don't think this will actually work. The `artifact-name` is for uploading a workflow artifact and attaching a release asset but the temporary file will be output to a somewhat randomized location. In the `sbom-action` there is a currently undocumented parameter: `output-file`, which would probably be what you want here.

Reply:
Alternately, it would be possible to set up a workflow using something like the [download workflow artifact](https://github.com/marketplace/actions/download-workflow-artifact) action, maybe.

The main thing is, I would like to have a workflow that supports:
* scanning an image/path once and generating an SBOM
* repeatedly scanning that same SBOM for vulnerabilities

The problem is, running vulnerability scanning in a manner like this with an SBOM reference needs access to a file, but a prior workflow will have dropped it and only supports possibly downloading artifacts currently, as I understand it.

Very happy to hear any other thoughts on this!

Reply:
yeah already found that `artifact-name` writes to `./random-name/file-name`, and I read somewhere about this undocumented parameter - but this certainly saves me a lot of debugging haha

I'll address it accordingly.

What's happing in this PR is a build-time check, which is only one part in a larger journey. I'd also like some other process scanning on a schedule, without requiring a new build of anything.

The deeper question I currently have is _where to store these generated SBOM_ 🤔 

- GitHub Artifacts as we create them currently expire after like 90days, and should go somewhere with a release (which maybe not something the user is doing in GitHub). 
- Plus, ideally the SBOM would be decoupled from it's parent so we can just scan it per regular interval; ie. place it in some centralized GH repo perhaps for easy/quick reference?
- OTOH it  is a part of the repo, so pushing the generated SBOM back into the repo keeps everything nicely in one place.

I'll hop on Slack 👍